docs/topic-guides/users.txt

```diff
@@ -3,31 +3,31 @@
 Users
 =====
 
-Determined is designed for teams of machine learning developers. By default,
-developers can access Determined without needing to authenticate; moreover,
-the objects (e.g., experiments and notebooks) created by one developer
+Determined is designed for teams of machine learning developers and objects
+(e.g., experiments and notebooks) created by one developer
 will be visible to all users of the system.
 
-While this is convenient for small teams, larger teams may prefer to
-create a Determined *user* account for each developer. After authenticating as
-a given user, the Determined objects created in that session will be
-associated with that user account. By default, the WebUI and CLI will
-only display objects created by the current user.
-
 Initially, a Determined installation has two users: ``admin`` and ``determined``.
 The ``determined`` user is the default account that is used for WebUI and CLI
-operations. If individual user accounts are not required, no additional
-configuration is needed. In this case, all assets created within Determined
-(experiments, commands, etc.) will be attributed to ``determined``.
+operations. If individual user accounts are not required, users can log in as the user
+``determined`` and no additional configuration is needed. In this case, all assets
+created within Determined (experiments, commands, etc.) will be attributed to
+``determined``.
 
 The ``admin`` user can be used to create users, change other users'
 passwords, and activate/deactivate users.
 
+While this is convenient for small teams, larger teams may prefer to
```

Comment:
question: Should we just change the guidance to have everyone make their own account? @vishnu2kmohan

Reply:
I talked with @vishnu2kmohan and I believe that's what he also thinks we should do. 

Do you think we should remove the mention that developers can _avoid_ making new users?

Reply:
I updated the messaging to encourage everyone to create users